src/data.js

```diff
@@ -572,6 +572,27 @@ module.exports = [
       'Gatsby',
     ],
   },
+  {
+    name: 'James Mathias',
+    description: 'Artist, Writer, & Outlaw',
+    url: 'https://leihu.com/uses',
+    twitter: '@jmathias',
+    emoji: '🍰',
+    country: '🇺🇸',
+    computer: 'apple',
+    phone: 'iphone',
+    tags: [
+      'Artist',
+      'Writer',
+      'Outlaw'
```

Comment:
You've got a comma missing at the end of this line

Reply:
Oh man, commas or semi-colons they're sneaky little devils!